python/semantic_kernel/prompt_template/handlebars/function_helpers.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import logging
+from typing import TYPE_CHECKING, Callable
+
+if TYPE_CHECKING:
+    from semantic_kernel.functions.kernel_arguments import KernelArguments
+    from semantic_kernel.functions.kernel_function import KernelFunction
+    from semantic_kernel.kernel import Kernel
+
+
+logger: logging.Logger = logging.getLogger(__name__)
+
+
+def create_helper_from_function(
```

Comment:
This same function will be useful for Jinja2 support. Can we move it to a more generic location instead of being in the handlebars dir?

Reply:
I've made the naming of the folder more generic, if the internal function signature is the same, then we should def reuse, for now it is in a file called handlebars_function_helpers.py so we can easily rename and adopt this function to be more generic, once you have the need for it!